build/Settings.cs

```diff
@@ -19,10 +19,10 @@ private static string config(string @default = null, [CallerMemberName] string k
                 : value;
         }
 
-        public const string DotnetIsolatedItemTemplatesVersion = "4.0.2057";
-        public const string DotnetIsolatedProjectTemplatesVersion = "4.0.2057";
-        public const string DotnetItemTemplatesVersion = "4.0.2057";
-        public const string DotnetProjectTemplatesVersion = "4.0.2057";
+        public const string DotnetIsolatedItemTemplatesVersion = "4.0.2213";
+        public const string DotnetIsolatedProjectTemplatesVersion = "4.0.2213";
```

Comment:
Just noting that this will probably need to be updated. For myget the latest version right now is 2216, and we don't know yet what the nuget version will be